tests/mir-opt/copy-prop/issue_107511.rs

```diff
@@ -1,9 +1,11 @@
-// skip-filecheck
 // EMIT_MIR_FOR_EACH_PANIC_STRATEGY
 //@ test-mir-pass: CopyProp
 
 // EMIT_MIR issue_107511.main.CopyProp.diff
 fn main() {
+    // CHECK-LABEL: fn main(
+    // CHECK-NOT: StorageLive(_16);
```

Comment:
Could you add debuginfo for `_16`, so we understand what happens?

Reply:
Added in https://github.com/rust-lang/rust/pull/135099/commits/430d88865386b007f00f27190960ae41befac686